scdl/scdl.py

```diff
@@ -489,7 +514,7 @@ def setMetadata(track, filename, album=None):
                 )
         else:
             logger.error('Artwork can not be set.')
-    audio.save()
+    audio.save(v2_version=3)
```

Comment:
Changed TagID to 2.3 instead of 2.4.
More stable and fix the date issue:
https://stackoverflow.com/questions/18026516/some-mutagen-tags-dont-work